runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectRunnerApiSurfaceTest.java

```diff
@@ -73,7 +73,8 @@ public void testDirectRunnerApiSurface() throws Exception {
             .pruningPattern("org[.]apache[.]beam[.].*IT")
             .pruningPattern("java[.]io.*")
             .pruningPattern("java[.]lang.*")
-            .pruningPattern("java[.]util.*");
+            .pruningPattern("java[.]util.*")
+            .pruningPrefix("org.apache.beam.repackaged");
```

Comment:
We shouldn't prune repackaged dependencies since this means that we will be allowing things like repackaged guava as part of our dependency surface.

Add a JIRA + TODO about figuring out why we are exposing this. The investigation should be a release blocker.